src/importer/defaults/generateDocumentPath.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+export default async function generateDocumentPath({
```

Comment:
```suggestion
export async function generateDocumentPath({
```

Reply:
While I agree I should not mix styles (see below), why should I get rid of the `default` ? Those modules represent just one function, I want to be able to do `import generateDocumentPath from 'generateDocumentPath.js'` which is nicer than `import { generateDocumentPath } from 'generateDocumentPath.js'` if the module exposes only one function. No ?

Reply:
Also, all the rest of the code use `default` exports.